src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -1,10 +1,12 @@
-﻿using System;
+﻿using Autodesk.DesignScript.Runtime;
+using Dynamo.Utilities;
+using DynamoUtilities;
+using Python.Runtime;
+using System;
 using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
-using Autodesk.DesignScript.Runtime;
-using Dynamo.Utilities;
-using Python.Runtime;
+using System.Reflection;
```

Comment:
@marimano Can you check your our coding standards and adjust your VS setting? https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards